api/controllers/UserController.ts

```diff
@@ -0,0 +1,43 @@
+import * as Package from 'pjson';
+import { Controller, Get, BaseRequest, BaseResponse, Post } from 'ts-framework';
+import {User} from '../models';
+import { UserService, ExtractService } from '../services/';
+
+@Controller('/user')
+export default class UserController {
+
+  @Get('/:id')
+  static async getUser(req: BaseRequest, res: BaseResponse) {
+    return res.success({
+      user: await UserService.getUser(req.params.id)
+    });
+  }
+
+  @Get('/:id/kyc')
+  static async getKycStatusForUser(req: BaseRequest, res: BaseResponse) {
+
+  }
+
+  @Get('/:id/extract')
+  static async getUserExtract(req: BaseRequest, res: BaseResponse) {
+    return res.success({
+      buys: await ExtractService.getBuyExtractForUser(req.params.id),
+      sells: await ExtractService.getSellExtractForUser(req.params.id)
```

Comment:
You can perform both operations in a single await (parallel execution) to reduce response time.

```typescript
const [buys, sells] = await Promise.all([
  ExtractService.getBuyExtractForUser(req.params.id),
  ExtractService.getSellExtractForUser(req.params.id)
]);

return res.success({ buys, sells });
```